### PR TITLE
Add command to display the current version of facerec

### PR DESCRIPTION
**Pull request by pulkitvyas08**
Solved #10 
Created the file "_version.py". The only file to modify for every new version

### CODE REVIEW

**Comment by rushabh-v** [comment]
The current version is `1.1`. I first decided to release it as `2.0` but finally released it as `1.1`. (There was a mistake in `setup.py`). The correct version can be found in `debian/changelog` at any point.

File: _version.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Store the version here so:
+# 1) we don't load dependencies by storing it in __init__.py
+# 2) we can import it in setup.py for the same reason
+# 3) we can import it into your module module
```
Comment (rushabh-v) [inline _version.py]: Rename this file to `config.py` and remove these comments.

**Review by rushabh-v — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -34,6 +34,7 @@ source ~/.bashrc
 | facerec disable | Temporarily disable facerec |
 | facerec remove | Completely remove the facerec  |
 | facerec --help | Get the CLI info |
+| facerec version | Display installed version of facerec |
```
Comment (rushabh-v) [inline README.md]: Keep `facerec version` before `facerec --help`. And change the command to `facerec --version`.

File: src/cli/cli.sh [hunk 3]
```diff
@@ -41,6 +41,10 @@ elif [ "$1" = "remove" ]; then
 elif [ "$1" = "--help" ]; then
     python3 /lib/Auth/Facerec/cli_info.py
 
+
+elif [ "$1" = "version" ]; then
```
Comment (rushabh-v) [inline src/cli/cli.sh]: `facerec --version`

File: src/cli/cli_info.py [hunk 4]
```diff
@@ -10,6 +10,7 @@
     ('facerec disable', 'Temporarily disable facerec. (preserve the setup)'),
     ('facerec remove', 'Completely remove the facerec.'),
     ('facerec --help', 'Get info of the CLI'),
+    ('facerec version', 'Display installed version of facerec'),
```
Comment (rushabh-v) [inline src/cli/cli_info.py]: ditto

**Comment by pulkitvyas08** [comment]
Resolved version issue in #10 and completed part of #11

File: _version.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
+__version__ = 1.1
+
```
Comment (rushabh-v) [inline _version.py]: import the config here and use `config.__version__`

File: config.py [hunk 6]
```diff
@@ -0,0 +1,6 @@
+from config import __version__
+
+name='facerec'
+version=__version__
+license='GPL-3'
+url='http://github.com/rushabh-v/linux_face_unlock'
```
Comment (rushabh-v) [inline config.py]: Remove `from config import __version__` line. And add spaces around operators. e.g. `=`

File: debian/install [hunk 7]
```diff
@@ -3,6 +3,7 @@ src/cli/. lib/Auth/Facerec/
 build.py lib/Auth/Facerec/
 comm_auth_orig.py lib/Auth/Facerec/
 Facerec lib/Auth/Facerec/
+config.py lib/Auth/Facerec
```
Comment (rushabh-v) [inline debian/install]: put `/` at the end.

File: setup.py [hunk 8]
```diff
@@ -1,5 +1,6 @@
 import os
 from setuptools import setup, find_packages
+from config import __version__
```
Comment (rushabh-v) [inline setup.py]: Just do `import config`. Use `config.__version__` here. And open one more PR to use everything else from the config file that can be used here (as that's not in the scope of this PR).

**Comment by rushabh-v** [comment]
I have enabled linter and CI checks for PRs!

File: setup.py (hunk 8 above)
Comment (pulkitvyas08) [reply setup.py]: Ok, I'll open a new PR

**Comment by rushabh-v** [comment]
Add `--version` to auto-completion

**Comment by pulkitvyas08** [comment]
Added --version to auto-completion. Anything more?

**Comment by rushabh-v** [comment]
> Added --version to auto-completion. Anything more?

No. 
Will merge on green!

**Comment by pulkitvyas08** [comment]
Awesome, thanks!

**Review by rushabh-v — APPROVED** [review]

**Comment by rushabh-v** [comment]
give @pulkitvyas08 25 points

**Pull request merged**